cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,21 @@
+class ProductsListPage {
+    private tShirtMenu: string;
+    private proceedToCheckout: string
+
+    constructor() {
+        /*With this selector suggested is possible to have the number 1 as a variable
+        to tests more items in the page*/
+        this.tShirtMenu = "[title='Add to cart'][data-id-product='1']";
+        this.proceedToCheckout = ".btn[title='Proceed to checkout']"
+    }
+
+    public goToTShirtMenu(): void {
+        cy.get(this.tShirtMenu).click({force: true, multiple: true})
```

Comment:
why it is needed the `{force: true, multiple: true}`

Reply:
this method should be in the MenuContentPage class

Reply:
Resolved